app/src/screens/MapScreen.js

```diff
@@ -88,10 +88,10 @@ function MapScreen(): React.Node {
   );
   const [satelliteView, setSatelliteView] = useState<boolean>(true);
   const [cacheMapDialogOpen, setCacheMapDialogOpen] = useState<boolean>(false);
-  const [filterMinRssi, setFilterMinRssi] = useState('');
-  const [filterMaxRssi, setFilterMaxRssi] = useState('');
-  const [filterMinHeight, setFilterMinHeight] = useState('10');
-  const [filterMaxHeight, setFilterMaxHeight] = useState('');
+  const [filterMinRssi, setFilterMinRssi] = useState<string>('');
```

Comment:
Tiny nit, but I prefer to use nulls instead of empty strings.  I would do something like this instead:
`const [filterMinRssi, setFilterMinRssi] = useState<string?>(null);`

Reply:
This was actually an unnecessary change that I forgot to undo. I was trying to figure out why there was a discrepancy in the filter code, and thought this was the cause.

The reason it's not nullable is because that actually results in potential missing edge cases in the filter code. It would start off null when there's nothing in the filter, and then not be null when a user tries to remove the filter altogether. Then, we'd have to handle both the null case, and the empty string case, which due to the unsafe null checks would require additional code.

Having these never be null makes more sense to me in this case - especially because we don't want to differentiate between when we programmatically instantiated vs when the user deleted the input.